monitoring/monitoring.go

```diff
@@ -68,32 +64,46 @@ type AlertSender interface {
 	SendAlert(alert cloudprotocol.AlertItem)
 }
 
+// Sender sends alerts to the cloud.
+type MonitoringSender interface {
+	SendMonitoringData(monitoringData cloudprotocol.MonitoringData) (err error)
+}
+
 // TrafficMonitoring interface to get network traffic.
 type TrafficMonitoring interface {
 	GetSystemTraffic() (inputTraffic, outputTraffic uint64, err error)
 	GetInstanceTraffic(instanceID string) (inputTraffic, outputTraffic uint64, err error)
 }
 
+// Monitoring configuration for system monitoring.
+type Monitoring struct {
+	MaxOfflineMessages int               `json:"maxOfflineMessages"`
```

Comment:
this parameter is not used. it will be a part of CM configuration

Reply:
Done